tools/ci_build/github/linux/docker/scripts/manylinux/install_centos.sh

```diff
@@ -19,9 +19,13 @@ if [ "$os_major_version" == "7" ]; then
     yum install -y dotnet-sdk-2.1
 fi
 
-yum install -y java-1.8.0-openjdk-devel
+if [ $USE_JDK11 == "1" ]; then
+    yum install -y java-11-openjdk-devel
```

Comment:
do we still have a need to not use JDK 11 in other builds?

Reply:
For java build for official packages other than Android AAR packages, I am not sure if using jdk11 is OK, maybe we can switch entirely to jdk11 after 1.8 release.
@Craigacp

Reply:
We can switch to Java 11 and target Java 8 for compatibility. The build.gradle is already setup to do that - https://github.com/microsoft/onnxruntime/blob/master/java/build.gradle#L81, but if the build JDK is always 11 (or greater) then we can simplify the build file too. That'll need a note somewhere in the readme and possibly a check in the cmake or build.gradle to give users a nice error message explaining what's going on.

Reply:
I thought the Java pipelines were switched over to use 11 by default anyway? I thought that some of them moved across, hence this issue - https://github.com/microsoft/onnxruntime/issues/6584.

Reply:
if we can use JDK 11 consistently and simplify things, should we just try to do that now? another benefit is that there could be fewer distinct build docker images to cache.

Reply:
Move to jdk11 entirely here for this docker image , however, since most of our CI pipelines are still using jdk8, which is out of scope of this change.